### DISCUSSION TITLE
🎉 Playground v2.5 addition is awesome

### DISCUSSION THREAD

**Discussion by BarrenWardo** [lllyasviel/Fooocus#3199] [Show and tell]
I've not been able to use Fooocus for a while now, but today I used it again and noticed that Playground v2.5 has been added. Honestly, the compositions are just superb! 🎉

This is just an appreciation post for all the devs involved in creating this amazing tool and model. All the images below are not cherry-picked. Kudos to the team! 🙌

![2024-06-27_18-16-37_9966](https://github.com/lllyasviel/Fooocus/assets/86141456/fd419093-6486-41e3-a7f1-2ff7381d07ae)
![2024-06-27_18-18-35_2430](https://github.com/lllyasviel/Fooocus/assets/86141456/4b8a9bed-b4b0-4d36-896c-93b30b713915)
![2024-06-27_18-22-36_4939](https://github.com/lllyasviel/Fooocus/assets/86141456/19f8b166-8a84-4855-a397-7dbac88d0701)
![2024-06-27_18-23-32_8820](https://github.com/lllyasviel/Fooocus/assets/86141456/3f3f3339-3f69-4fcb-82b0-971a845c7c14)
![2024-06-27_18-28-35_2638](https://github.com/lllyasviel/Fooocus/assets/86141456/ee149a2b-408a-4058-adc6-730fba2e05a5)
![2024-06-27_18-07-05_9389](https://github.com/lllyasviel/Fooocus/assets/86141456/8db3663b-7c6e-48c8-a3a2-4b2305cbf534)

**Comment by mashb1t** [comment]
@BarrenWardo fyi moved from General to Show & Tell
